internal/backends/error.go

```diff
@@ -40,6 +40,9 @@ const (
 	ErrorCodeCollectionAlreadyExists
 
 	ErrorCodeInsertDuplicateID
+
+	ErrorCodeIndexDoesNotExist
+	ErrorCodeIndexInvalidOptions
```

Comment:
Please remove both – we don't do validation when we create indexes

Reply:
Then I need to modify tests again to have `failsForSQLite` for particular cases. Can we release it as is and cleanup after the release?